src/arviz_plots/plots/pair_plot.py

```diff
@@ -0,0 +1,495 @@
+    labeller : labeller, optional
+    aes_by_visuals : mapping, optional
+        Mapping of visuals to aesthetics that should use their mapping in `plot_matrix`
+        when plotted. Valid keys are the same as for `visuals`.
+        By default, there are no aesthetic mappings at all
+    visuals : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * scatter -> passed to :func:`~.visuals.scatter_couple`
+        * divergence -> passed to :func:`~.visuals.scatter_couple`. Defaults to False.
+        * dist -> depending on the value of `marginal_kind` passed to:
+
+          * "kde" -> passed to :func:`~arviz_plots.visuals.line_xy`
+          * "ecdf" -> passed to :func:`~arviz_plots.visuals.ecdf_line`
+          * "hist" -> passed to :func: `~arviz_plots.visuals.hist`
+
+        * credible_interval -> passed to :func:`~arviz_plots.visuals.line_x`
+        * point_estimate -> passed to :func:`~arviz_plots.visuals.scatter_x`
+        * point_estimate_text -> passed to :func:`~arviz_plots.visuals.point_estimate_text`
+        * label -> passed to :func:`~.visuals.label_plot`
+        * xlabel -> passed to :func:`~.visuals.labelled_x`
+        * ylabel -> passed to :func:`~.visuals.labelled_y`
+        * diag_xlabel -> passed to :func:`~.visuals.labelled_x` for the diagonal plots
+        * remove_axis -> not passed anywhere, can only be ``True`` to  call this function
+
+    stats : mapping, optional
+        Valid keys are:
+
+        * dist -> passed to kde, ecdf, ...
+        * credible_interval -> passed to eti or hdi
+        * point_estimate -> passed to mean, median or mode
+
+    **pc_kwargs
+        Passed to :class:`arviz_plots.PlotMatrix`
+
+
+    Returns
+    -------
+    PlotMatrix
+
+    Examples
+    --------
+    Default plot_pair
+
+    .. plot::
+        :context: close-figs
+
+        >>> from arviz_plots import plot_pair, style
+        >>> style.use("arviz-variat")
+        >>> from arviz_base import load_arviz_data
+        >>> dt = load_arviz_data('centered_eight')
+        >>> plot_pair(
+        >>>     dt,
+        >>>     var_names=["mu", "tau"],
+        >>>     visuals={"divergence": True},
+        >>>     marginal=True,
+        >>>     marginal_kind="hist",
+        >>> )
+
+    .. minigallery:: plot_pair
+
+    """
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if isinstance(sample_dims, str):
+        sample_dims = [sample_dims]
+    if visuals is None:
+        visuals = {}
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+
+    if labeller is None:
+        labeller = BaseLabeller()
+    if backend is None:
+        if plot_matrix is None:
+            backend = rcParams["plot.backend"]
+        else:
+            backend = plot_matrix.backend
+
+    if marginal_kind is None:
+        marginal_kind = rcParams["plot.density_kind"]
```

Comment:
This is not used anywhere, so we can leave the substitution of None for the rcParam to `plot_dist`